src/test/java/org/kainos/groupA/integration/JobRoleIntegrationTest.java

```diff
@@ -0,0 +1,29 @@
+package org.kainos.groupA.integration;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.kainos.groupA.GroupAApplication;
+import org.kainos.groupA.GroupAConfiguration;
+import org.kainos.groupA.models.JobRole;
+
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobRoleIntegrationTest {
+    static final DropwizardAppExtension<GroupAConfiguration> APP = new DropwizardAppExtension<>(
+            GroupAApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void getJobRoles_shouldReturnListOfJobRoles() {
+       List<JobRole> response = APP.client().target("http://localhost:8080/api/job-roles")
+               .request()
+               .get(List.class);
+        Assertions.assertTrue(response.size() > 0);
```

Comment:
can you assert that the first element in the list has a role name that is not null?